client/vault/Makefile

```
@@ -0,0 +1,85 @@
+ifeq ($(OS),Windows_NT)
+	DYLIB := ../../target/$(PROFILE)/$(LIB_NAME).dll
+else
+	UNAME_S := $(shell uname -s)
+	ifeq ($(UNAME_S),Darwin)
+		DYLIB := ../../target/$(PROFILE)/lib$(LIB_NAME).dylib
+	else
+		DYLIB := ../../target/$(PROFILE)/lib$(LIB_NAME).so
+	endif
+endif
+
+.PHONY: gen python smoke-test smoke-test-view smoke-test-full
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)